src/Bridge/NelmioApiDoc/Parser/ApiPlatformParser.php

```diff
@@ -171,7 +174,8 @@ private function getPropertyMetadata(ResourceMetadata $resourceMetadata, string
                 ($propertyMetadata->isReadable() && self::OUT_PREFIX === $io) ||
                 ($propertyMetadata->isWritable() && self::IN_PREFIX === $io)
             ) {
-                $data[$propertyName] = $this->parseProperty($resourceMetadata, $propertyMetadata, $io, null, $visited);
+                $normalizedPropertyName = null !== $this->nameConverter ? $this->nameConverter->normalize($propertyName) : $propertyName;
```

Comment:
`$this->nameConverter ? $this->nameConverter->normalize($propertyName) : $propertyName;`? It doesn't hurt here because `$this->nameConverter` can only be null or a reference.